packages/language-server/src/plugins/svelte/features/getCompletions.ts

```diff
@@ -54,9 +54,13 @@ function getCompletionsWithRegardToTriggerCharacter(
 
     if (triggerCharacter === '#') {
         return createCompletionItems([
-            { tag: 'if', label: 'if' },
-            { tag: 'each', label: 'each' },
-            { tag: 'await', label: 'await' },
+            { tag: 'if', label: 'if', insertText: 'if $1}\n\t$2\n{/if' },
+            { tag: 'each', label: 'each', insertText: 'each $1 as $2}\n\t$3\n{/each' },
+            {
+                tag: 'await',
+                label: 'await',
+                insertText: 'await $1}\n\t$2\n{:then $3} \n\t$4\n{/await',
```

Comment:
Altough this is super nice, there is another possibility to write the `await` construct: `{#await expression then name}...{/await}`. Maybe add another `await` completion with that other `insertText`?